x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts

```diff
@@ -8,61 +8,119 @@
 import { EntityDefinition, entityDefinitionSchema } from '@kbn/entities-schema';
 import { BUILT_IN_ID_PREFIX } from './constants';
 
-export const builtInServicesEntityDefinition: EntityDefinition = entityDefinitionSchema.parse({
-  id: `${BUILT_IN_ID_PREFIX}services`,
-  name: 'Services from logs',
-  type: 'service',
-  managed: true,
-  indexPatterns: ['logs-*', 'filebeat*'],
-  history: {
-    timestampField: '@timestamp',
-    interval: '1m',
-  },
-  latest: {
-    lookback: '5m',
-  },
-  identityFields: ['service.name', { field: 'service.environment', optional: true }],
-  displayNameTemplate: '{{service.name}}{{#service.environment}}:{{.}}{{/service.environment}}',
-  metadata: [
-    'data_stream.type',
-    'service.instance.id',
-    'service.namespace',
-    'service.version',
-    'service.runtime.name',
-    'service.runtime.version',
-    'service.node.name',
-    'service.language.name',
-    'agent.name',
-    'cloud.provider',
-    'cloud.instance.id',
-    'cloud.availability_zone',
-    'cloud.instance.name',
-    'cloud.machine.type',
-    'host.name',
-    'container.id',
-  ],
-  metrics: [
-    {
-      name: 'logRate',
-      equation: 'A / 5',
-      metrics: [
-        {
-          name: 'A',
-          aggregation: 'doc_count',
-          filter: 'log.level: *',
-        },
-      ],
+export const builtInServicesFromLogsEntityDefinition: EntityDefinition =
+  entityDefinitionSchema.parse({
+    id: `${BUILT_IN_ID_PREFIX}services_from_logs`,
```

Comment:
nit: The id is a bit misleading as the services will be from logs and apm based on indexPatterns.

Reply:
I'll update

Reply:
Suggested fix here https://github.com/elastic/kibana/pull/187021/commits/a891fb2ff5f288ff4af1b2b1d29ee5f641376186